--- anteater-0.4.5/configure.in~	2003-11-04 11:13:52.000000000 +0100
+++ anteater-0.4.5/configure.in	2010-09-21 15:37:11.000000000 +0200
@@ -54,7 +54,7 @@
 dnl -------------------------------------------------------------------------
 AC_MSG_CHECKING(for namespace support)
 AC_TRY_RUN([namespace testns{int givemenull(void){return 0;}}
-int main(){exit(testns::givemenull());}]
+int main(){return testns::givemenull();}]
 ,AC_MSG_RESULT(yes),AC_MSG_RESULT(no)
 AC_MSG_ERROR([Namespaces are required to compile this package.
 Please update your compiler. In the most cases this is not very difficult.
--- anteater-0.4.5/configure.in~	2010-09-21 15:38:43.000000000 +0200
+++ anteater-0.4.5/configure.in	2010-09-21 15:39:47.000000000 +0200
@@ -64,7 +64,7 @@
 AC_MSG_CHECKING(for pointer to classmembers)
 AC_TRY_RUN([class ctest{public:int run(void){return 0;};};
 int main(){int(ctest::*cmp)(void);
-cmp=&ctest::run;ctest ni;exit((ni.*cmp)());}],AC_MSG_RESULT(yes),AC_MSG_RESULT(no)
+cmp=&ctest::run;ctest ni;return (ni.*cmp)();}],AC_MSG_RESULT(yes),AC_MSG_RESULT(no)
 AC_MSG_ERROR([Pointer to classmembers are required to compile this package.
 Please update your compiler. In the most cases this is not very difficult.
 Have a look at the README file in this distribution.]),AC_MSG_WARN(
